### PR TITLE
DOC Add graphql rebuild section

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-graphql/issues/500

### CODE REVIEW

File: en/04_Changelogs/4.13.0.md
Comment (ChloeSartorelli) [inline en/04_Changelogs/4.13.0.md]:
```suggestion
- For `silverstripe/graphql` v4 an additional fallback has been added  to rebuild the schema if a file is missing. There is a configurable minimum interval between this type of rebuild occuring, which is set to 15 minutes by default. This type of rebuild can also be disabled via configuration. These configuration options are available on [`SilverStripe\GraphQL\Schema\Storage\AbstractTypeRegistry`](api:SilverStripe\GraphQL\Schema\Storage\AbstractTypeRegistry).
```

We also normally have examples of setting the config in yaml code blocks for this sort of thing - I wonder if it's worth giving this it's own heading?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/04_Changelogs/4.13.0.md
Comment (emteknetnz) [reply en/04_Changelogs/4.13.0.md]: Updated

Comment (ChloeSartorelli) [reply en/04_Changelogs/4.13.0.md]: The recommended changes to the text have not been made

File: en/04_Changelogs/4.13.0.md
Comment (ChloeSartorelli) [inline en/04_Changelogs/4.13.0.md:80]:
```suggestion
### GraphQL v4 additional fallback
```
I won't hold back merging for this, but as I've mentioned a few times before you don't need to add the anchor if it's identical to the default anchor. Applies to the one below as well.

File: en/04_Changelogs/4.13.0.md
Comment (emteknetnz) [reply en/04_Changelogs/4.13.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
Looks good, holding off merge until the code changes are merged.

**Comment by GuySartorelli** [comment]
Oops forgot to merge this

**Pull request merged**
